packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.ts

```diff
@@ -682,6 +704,7 @@ export const InstanceProperties = {
   codePoints: define("instance/code-points", ["esnext.string.code-points"]),
   concat: define("instance/concat", ["es.array.concat"], undefined, ["String"]),
   copyWithin: define("instance/copy-within", ["es.array.copy-within"]),
+  demethodize: define("instance/demethodize", ["esnext.function.demethodize"]),
```

Comment:
I have a question. This proposal has never been presented to TC39, what's the core-js policy regarding when to implement a proposal? If a TC39 delegate creates a repository for it?

Reply:
It was presented in the scope of another proposal, https://github.com/js-choi/proposal-function-demethodize/blob/main/HISTORY.md - it was decided to extract it.

Reply:
> If a TC39 delegate creates a repository for it?

https://github.com/js-choi/proposal-function-demethodize

Reply:
Be forewarned that this name isn’t likely to last.

Reply:
Agree. But in any case, it's better than outdated.